modules/example/src/smithy4s/example/LowHigh.scala

```diff
@@ -20,12 +25,12 @@ sealed abstract class LowHigh(_value: String, _name: String, _intValue: Int, _hi
 object LowHigh extends Enumeration[LowHigh] with ShapeTag.Companion[LowHigh] {
   val id: ShapeId = ShapeId("smithy4s.example", "LowHigh")
 
-  val hints: Hints = Hints(
-    smithy.api.Enum(List(smithy.api.EnumDefinition(value = smithy.api.NonEmptyString("Low"), name = Some(smithy.api.EnumConstantBodyName("LOW")), documentation = None, tags = None, deprecated = None), smithy.api.EnumDefinition(value = smithy.api.NonEmptyString("High"), name = Some(smithy.api.EnumConstantBodyName("HIGH")), documentation = None, tags = None, deprecated = None))),
-  )
+  val hints: Hints = Hints.empty
```

Comment:
**issue:** this doesn't look right, IMO we should keep the traits originally present in the model, regardless if they're duplicated in the Enumeration schema

Reply:
I guess there's no "right thing to do", it'll be a judgement call - do we try to do what Smithy does in `upgradeEnums` (which is an optional transformation that uses the code path you mentioned), or do we try to make our representation as close to the original as possible?

I think for support of Smithy 1.x, we usually did the latter.

Reply:
Okey, I will keep enum trait information in generated hints.